### ISSUE TITLE
Add `--export` flag

### ISSUE THREAD

**Issue by mmaelicke** [VForWaTer/metacatalog#141]
After #140 is implemented, the ResultSet lists can wrap anything found by `api.find_entry` or `api.find_group` and export it directly

**Comment by mmaelicke** [comment]
This is already done for the FIND CLI
I don't think an extra export cli is needed at this point
